SPEC.md

```diff
@@ -543,6 +543,8 @@ Accessible at `$AC_METADATA_URL/acMetadata/v1/pod/hmac`
 
 ### AC Name Type
 
+Field `acName`
```

Comment:
Hrmmm but still, the AC Name type is used much more broadly than for the app name..